superchain/superchain.go

```diff
@@ -485,7 +574,7 @@ type SuperchainConfig struct {
 	SuperchainConfigAddr *Address `yaml:"superchain_config_addr,omitempty"`
 
 	// Hardfork Configuration. These values may be overridden by individual chains.
-	hardForkDefaults HardForkConfiguration
+	HardForkDefaults HardForkConfiguration
```

Comment:
Was it necessary to make this an exported field? The intention was to make this unexported, to force consumers to read the overriden values from each chain.

Reply:
Originally was reading these defaults from within the `add-chain` module. But refactored and am using `rollupConfig.SetDefaultHardforkTimestampsToNil` method to read the defaults so can change `HardForkDefaults` be to unexported